src/main/java/net/minecraftforge/common/config/ConfigManager.java

```diff
@@ -117,7 +118,7 @@ public static void loadData(ASMDataTable data)
         }
     }
 
-    public static void load(String modid, Config.Type type)
+    public static void sync(String modid, Config.Type type)
     {
```

Comment:
Note, it may be worth making the internal function that walks the config class to build the tree private and pass in the 'direction' it's going. 
```
private static void walkConfig(String modid, String category, Configuration cfg, Class<?> ftype, Field f, Object instance, boolean isCreation, boolean toObject){
...
  if (toObject) set(field, cfg.get())
  else              cfg.set(name, getValue(field))
  if (isCreate)
    cfg.getProp(name).setDefault().setRange().setComment()...
...
}
```
This way you don't have to duplicate the bulk of the function which is walking the class tree.